### ISSUE TITLE
Wait time is not working

### ISSUE THREAD

**Issue by Kongux** [oRastor/fut22-web-app#89]
Hi oRastor, 

the random wait time between the searches is not working. The bot ist not waiting between the searchs. He is directly going on after finding some results. Can you fix it? 
Best Regard 
K0ngux

**Comment by oRastor** [comment]
It works, you must specify two numbers splitted by "-", like "7-12"

**Comment by Kongux** [comment]
Yes i specified two numbers. But he continues searching every second. I am using a Macbook. Is this maybe the Reason? In the past i never had Problems…
